### DISCUSSION TITLE
Spell Checking

### DISCUSSION THREAD

**Discussion by sir-adam-smith** [fleather-editor/fleather#109] [Q&A] [answered]
This is such an awesome package, you guys did a great job!

Just wanted to find out if there is any way to also have spell checking on the text somehow?

Regards
Adam

**Comment by amantoux** [comment, marked as answer]
It's not in our short term plans but we're open to proposals.

We can provide an API that takes care of the squiggles as a minimum

**Reply by sir-adam-smith** [reply]
This would be awesome! Thanks!
